### ISSUE TITLE
Recruit external contributors to WCAG 2.2 video and create assets for video

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4238]
## Brief
By the end of the cycle we should have:
- content review of draft script (time boxed to 2 hours)
- contacted external contributors to the video, agreed on content and arranged a date for recording
- identified and created all graphic design elements required in the video

## Epic lead
Kelly

## Driving role(s)
Charlotte

## Supporting roles
Trang, Hazal, content specialist

## Needs awareness
DSP comms team

```[tasklist]
### Tasks
- [ ] https://github.com/alphagov/govuk-design-system/issues/4219
- [ ] https://github.com/alphagov/govuk-design-system/issues/4286
- [ ] https://github.com/alphagov/govuk-design-system/issues/4287
- [ ] https://github.com/alphagov/govuk-design-system/issues/4288
```
